packages/image-block/src/ImageBlock.tsx

```diff
@@ -56,6 +57,12 @@ export const ImageBlock = withAttachmentsProvider(({ appBridge }: BlockProps) =>
     const { blockAssets, deleteAssetIdsFromKey, updateAssetIdsFromKey } = useBlockAssets(appBridge);
     const image = blockAssets?.[IMAGE_ID]?.[0];
     const attachmentCount = blockAssets[ATTACHMENTS_ASSET_ID]?.length || 0;
+    const isImageDownloadable = isDownloadable(
```

Comment:
add here also the check if its a protected image, 

If its protected always false, otherewise determine based on the `isDownloadable`

We will move that logic in the long run into the `guideline-blocks-settings` but lets wait, so its not becoming blocking

Reply:
Yep, updated!